.github/workflows/linter.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Lint
+
+on: 
+  push: null
+  pull_request: null
+
+permissions: { }
+
+jobs:
+  build:
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
only runs in ubuntu?

Reply:
thank youfor honest feedback i updated the lint now it is able to run on across all platform and also i added the functionality  to lint only newly pushed changes